torch/_torch_docs.py

```diff
@@ -5329,7 +5329,7 @@ def merge_dicts(*dicts):
 Closeness is defined as:
 
 .. math::
-    \lvert \text{input} - \text{other} \rvert \leq \texttt{atol} + \texttt{rtol} \times \lvert \text{other} \rvert
+    \lvert \text{input}_i - \text{other}_i \rvert \leq \texttt{atol} + \texttt{rtol} \times \lvert \text{other}_i \rvert \quad \forall i
```

Comment:
Can we remove `\forall` ? 

I do agree that it makes sense to add the subscript, e.g. to match  https://pytorch.org/docs/stable/generated/torch.add.html#torch.add and other element-wise type operations.

But for me that `forall` seems to indicate that the function will return True/False based on whether EVERY element-wise comparison returned true.